docs/DeveloperGuide.md

```diff
@@ -312,11 +312,23 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Non-Functional Requirements
 
-1.  Should work on any _mainstream OS_ as long as it has Java `11` or above installed.
-2.  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
-3.  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
+1. Technical Requirements:
+    1. The application should work on any _mainstream OS_, such as Windows, Linux, and macOS, as long as it has Java `11` or above installed.
+    2. The application should be compatible with both _32-bit_ and _64-bit_ environments.
+2. Performance Requirements:
+    1. Should be able to hold up to _1000 patients and nurses_ without noticeable sluggishness in performance for typical usage.
```

Comment:
Maybe here add one more 0, here 10000? but I not sure about it too cuz seems like it should more since hospital can have a lot of nurses/ patients ....

Reply:
That's true, I'll increase it.